seq2science/rules/peak_calling.smk

```diff
@@ -358,7 +358,8 @@ if "biological_replicates" in samples:
             params:
                 rank=lambda wildcards: "--rank 13" if wildcards.peak_caller == "hmmratac" else "",
                 nr_reps=lambda wildcards, input: len(input),
-                reps=lambda wildcards, input: input,  # help resolve changes in input files
+                reps=lambda wildcards, input: input,  # help resolve changes in input files,
```

Comment:
optional: `reps=lambda wildcards, input: input, # help resolve changes in input files,` can go. `wildcards, input` already checked in the param above

Reply:
I'm not sure. Since the one before only checks the length of the input, and not the actual input